libs/execution_context/unit/execution_context_unit_test.cpp

```diff
@@ -91,7 +91,11 @@ typedef class _ebpf_async_wrapper
 class _ebpf_core_initializer
 {
   public:
-    _ebpf_core_initializer() { REQUIRE(ebpf_core_initiate() == EBPF_SUCCESS); }
+    void
+    initialize()
```

Comment:
is there a scenario where we _do not_ want to call `initialize()`?

Reply:
No. but it cannot be safely called in the constructor, as explained in the PR description.